pubsublite/driver/driver.go

```diff
@@ -0,0 +1,9 @@
+package driver
+
+import "context"
+
+type Pubsubber interface {
+	Publish(ctx context.Context, topic string, msg []byte) error
```

Comment:
What about message metadata?

Reply:
It's not supported as a first-class concept in any of the at-most-once providers that I know of.  

NATS: https://github.com/nats-io/go-nats/blob/f3379e01130f595688283acd70cdb1fff65d9a4d/nats.go#L429:6

Redis: https://github.com/go-redis/redis/blob/bd542089bb6e776e6fced5038edac8a0f526aa53/pubsub.go#L273:6

ZeroMQ: https://github.com/pebbe/zmq4/blob/7a493a642e7acbd03045d4f5fe9516908a62f86f/zmq4.go#L983